tests/nested/manual/core20-early-config/install

```diff
@@ -3,4 +3,8 @@
 # rsyslog.service is expected to be a symlink at this point, and apparmor
 # doesn't control readlink, so this should work even without devmode or
 # system-files plug.
-readlink /etc/systemd/system/rsyslog.service > "$SNAP_COMMON"/debug.txt 2>&1 || true
+RSYSLOG=$(readlink /etc/systemd/system/rsyslog.service)
+echo "rsyslog symlink: $RSYSLOG" > "$SNAP_COMMON"/debug.txt 2>&1 || true
+
+BACKLIGHT=$(readlink /etc/systemd/system/systemd-backlight@.service)
+echo "backlight symlink: $BACKLIGHT" >> "$SNAP_COMMON"/debug.txt 2>&1 || true
```

Comment:
What is the 2>&1 redirect for? Echo doesn't write anything to stderr.

Reply:
Ah, indeed, thanks, it was a "leftover" from readlink, not really needed, will remove.